(dp1
S'output'
p2
S"<type 'exceptions.NameError'> global name 'organizar_datos_usuario' is not defined"
p3
sS'layer'
p4
S'/home/juliana/web2py/applications/red_social/controllers/perfil.py'
p5
sS'code'
p6
S'# -*- coding: utf-8 -*-\n\n@auth.requires_login() \ndef index():\n    return organizar_datos_usuario(auth.user)\n\ndef downloads():\n     return response.download(request, db)\n\n@auth.requires_login()\ndef ver():\n    id = request.args[0]\n    usuario = db.auth_user(id)\n    return __organizar_datos_usuario(usuario)\n\ndef __organizar_datos_usuario(usuario):\n    nombre_completo = usuario.first_name + " " + usuario.last_name\n    ruta_imagen =  URL(\'downloads\',args = usuario.foto)\n    return dict(nombre_completo = nombre_completo, \n                amigos = encuentra_mis_amigos(usuario.id),\n                ruta_imagen = ruta_imagen)\n\nresponse._vars=response._caller(index)\n'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/juliana/web2py/gluon/restricted.py", line 227, in restricted\n    exec ccode in environment\n  File "/home/juliana/web2py/applications/red_social/controllers/perfil.py", line 23, in <module>\n  File "/home/juliana/web2py/gluon/globals.py", line 412, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/juliana/web2py/gluon/tools.py", line 3774, in f\n    return action(*a, **b)\n  File "/home/juliana/web2py/applications/red_social/controllers/perfil.py", line 5, in index\n    return organizar_datos_usuario(auth.user)\nNameError: global name \'organizar_datos_usuario\' is not defined\n'
p11
s.